cmd/vulcan-aws-trusted-advisor/main.go

```diff
@@ -131,15 +121,35 @@ func scanAccount(opt options, target, assetType string, logger *logrus.Entry, st
 	if err != nil {
 		return err
 	}
-	creds, err := getCredentials(assumeRoleEndpoint, parsedARN.AccountID, role, logger)
-	if err != nil {
-		return err
+	var cfg aws.Config
```

Comment:
I think it would be better to move the reachability test after the get credentials part of the code.
Also validate that the account id you have access to is the target one.